src/main/java/uk/gov/hmcts/reform/sscs/model/service/hearingvalues/ServiceHearingValues.java

```diff
@@ -28,7 +29,7 @@ public class ServiceHearingValues {
     private String hearingPriorityType;
     private Integer numberOfPhysicalAttendees;
     private boolean hearingInWelshFlag;
-    private List<HearingLocation> hearingLocations;
+    private List<HearingLocations> hearingLocations;
```

Comment:
Minor: Can we change this class to HearingLocation as it only contains one location?